lld/test/ELF/invalid/x86-64-tls-reloc.s

```diff
@@ -0,0 +1,36 @@
+# REQUIRES: x86
```

Comment:
when testing a specific feature, we place negative tests beside the positive tests.

Perhaps move this to ELF/x86-64-tls-ie-err.s

Reply:
Moved to ELF/x86-64-tls-ie-err.s.